configs/experiment/conv3d_sat_nwp.yaml

```diff
@@ -8,7 +8,7 @@ defaults:
   - override /model: conv3d_sat_nwp.yaml
   - override /datamodule: ocf_datapipes.yaml
   - override /callbacks: default.yaml
-  - override /logger: neptune.yaml
+#  - override /logger: neptune.yaml
```

Comment:
your not using neptune? are you using wandb?

Reply:
Yeh, I made the switch to wandb